bluesky/callbacks/core.py

```diff
@@ -143,6 +115,31 @@ def get_obj_fields(fields):
     return string_fields
 
 
+# deprecate callbacks moved to mpl_plotting ----------------------------------
+
+def _deprecate_import_name(name):
+    wmsg = (
```

Comment:
I think we have to remove this warning. Won't it be tripped at import time every time when the function is called just below? It's not so important to have it, anyway.

Reply:
Yes, indeed it is called at the import time for 6 times. I'll remove it.